app/controllers/close_contacts_controller.rb

```diff
@@ -37,14 +37,14 @@ def update
     redirect_to(root_url) && return if patient_id.nil?
```

Comment:
this and the line about it are no longer necessary, please remove in update and create